Environments update (#2777)
fishtown-analytics/docs.getdbt.com: website/docs/docs/get-started/develop-in-the-cloud.md

:::

In order to start experiencing the great features of the Cloud IDE, you need to first set up a [dbt Cloud development environment](/docs/collaborate/environments/dbt-cloud-environments) and [developer credentials](#developer-credentials). If you're creating a new project, you will automatically configure this during the project setup. 

Review the following steps to set up your developer credentials:

### Developer credentials

The IDE uses developer credentials to connect to your data platform. These developer credentials should be specific to your user and they should *not* be super user credentials or the same credentials that you use for your production deployment of dbt.

Follow the below steps to set up your developer credentials:

> 1. In the upper right corn, click on gear icon -> **Profile Settings** -> [**Credentials**](https://cloud.getdbt.com/next/settings/profile#credentials).

Reviewer: ```suggestion
1. Go to your [**Credentials**](https://cloud.getdbt.com/next/settings/profile#credentials) under the **Your Profile** settings.
```

Reviewer: sugg:
- more concise 
- in docs, user can just click on link (unlike the links for **Account Settings** since they're unique to the user account). for this, user doesn't need to click on gear icon
